core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceEditorDirective.js

```diff
@@ -0,0 +1,162 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the a multiple choice question in the simple
+ * editor.
+ */
+
+oppia.directive('multipleChoiceEditor', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      getStateName: '&stateName',
+      getCustomizationArgs: '&customizationArgs',
+      getAnswerGroups: '&answerGroups',
+      getRawDefaultOutcome: '&defaultOutcome',
+      saveCustomizationArgs: '&',
+      saveAnswerGroups: '&',
+      saveDefaultOutcome: '&'
+    },
+    templateUrl: 'simpleEditorQuestions/MultipleChoiceInput',
+    controller: [
+      '$scope', '$timeout', 'focusService', 'SimpleEditorQuestionsDataService',
+      'explorationStatesService',
+      function(
+          $scope, $timeout, focusService, SimpleEditorQuestionsDataService,
+          explorationStatesService) {
+        $scope.questionId = Math.random().toString(36).slice(2);
```

Comment:
Yes, if it's repeated. However, I think I might change this to be done in the body directive and passed in, since it's the same for all the individual question editors. I think it's probably still worth adding a service, too; will do these things in a follow-up commit.